DIRECTORY.md

```diff
@@ -132,6 +132,7 @@
   * [Durand Kerner Roots](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/numerical_methods/durand_kerner_roots.cpp)
   * [False Position](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/numerical_methods/false_position.cpp)
   * [Gaussian Elimination](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/numerical_methods/gaussian_elimination.cpp)
+  * [Golden Search Extrema](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/numerical_methods/golden_search_extrema.cpp)
```

Comment:
AFAIK, the GitHub [workflow](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/.github/workflows/awesome_workflow.yml#L59) will re-add this algorithm here.

Reply:
No, it would not